assets/training/distillation/components/data_generation/src/generate_data.py

```diff
@@ -589,13 +607,15 @@ def data_import(args: Namespace):
     request_batch_size = args.request_batch_size
     min_endpoint_success_ratio = args.min_endpoint_success_ratio
     enable_cot_str = args.enable_chain_of_thought
+    enable_cod_str = args.enable_chain_of_density
     data_generation_task_type = args.data_generation_task_type
 
     # validate file formats
     _validate_file_paths_with_supported_formats([args.train_file_path, args.validation_file_path])
     logger.info("File format validation successful.")
 
     enable_cot = True if enable_cot_str.lower() == "true" else False
+    enable_cod = True if enable_cod_str.lower() == "true" else False
```

Comment:
A scalable way to handle this can be to have a flag "enhancement_strategy" which takes enum values in ["cod_for_summarization", "cot_for_qna"].. let me know what you folks think.